packages/lesswrong/server/vulcan-lib/apollo-ssr/renderPage.tsx

```diff
@@ -135,7 +135,7 @@ export const renderWithCache = async (req: Request, res: Response, user: DbUser|
   }
 };
 
-export function getThemeOptionsFromReq(req: Request, user: DbUser|null) {
+export const getThemeOptionsFromReq = (req: Request, user: DbUser|null): AbstractThemeOptions => {
   const themeCookie = getCookieFromReq(req, "theme");
```

Comment:
JP can't figure something out, that's probably not from this PR:

How does the client know, in javascript, what the theme is? The only time I see us reading from the theme cookie is on the server.

Reply:
We embed the value as `window.themeOptions` during SSR at line 306 of `apolloServer.ts`:
```
const themeOptionsHeader = embedAsGlobalVar("themeOptions", themeOptions);
```